<launch>
    <arg name="input_depth_image_topic" default="/camera/aligned_depth_to_color/image_raw"/>
    <arg name="camera_info_topic" default="/camera/aligned_depth_to_color/camera_info" />
    <arg name="bounding_boxes_topic" default="/cleanbot_2022/bounding_boxes" />


    <node name="object_tracker" pkg="cleanbot_2022_bringup" type="object_tracker.py" output="screen">
        <param name="input_depth_image" value="$(arg input_depth_image_topic)" />
        <param name="camera_info" value="$(arg camera_info_topic)" />
        <param name="bounding_boxes" value="$(arg bounding_boxes_topic)" />
    </node>
</launch>
